src/components/Content/Content.tsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[],
+  filter: string,
+  onFilterChange: (arg: string) => void,
+}
+export const Content: React.FC<Props> = ({
+  todos,
+  filter,
+  onFilterChange,
+}) => {
+  const handleShowAllTodo = () => {
+    onFilterChange('all');
+  };
+
+  const handleShowActiveTodo = () => {
+    onFilterChange('active');
+  };
+
+  const handleShowCompletedTodo = () => {
+    onFilterChange('completed');
+  };
+
+  const activeTodo = todos.filter(todo => !todo.completed);
+
+  return (
+    <div className="todoapp__content">
+      <header className="todoapp__header">
+        {/* this buttons is active only if there are some active todos */}
+        <button
+          type="button"
+          className="todoapp__toggle-all active"
+          aria-label=""
+        />
+
+        {/* Add a todo on form submit */}
+        <form>
+          <input
+            type="text"
+            className="todoapp__new-todo"
+            placeholder="What needs to be done?"
+          />
+        </form>
+      </header>
+
+      {todos.length > 0 && (
+        <>
+          <section className="todoapp__main">
+            {/* This is a completed todo */}
+            {todos.map(todo => {
+              const { id, completed, title } = todo;
```

Comment:
can we remove `return` keyword?
```suggestion
            {todos.map({id, completed, title}=> {
```